deploy/consts.ts

```diff
@@ -24,11 +20,7 @@ export const L1_ADDRESS_MAP: { [key: number]: { [contractName: string]: string }
     polygonERC20Predicate: "0x40ec5B33f54e0E8A33A975908C5BA1c14e5BbbDf",
     polygonRegistry: "0x33a02E6cC863D393d6Bf231B697b82F6e499cA71",
     polygonDepositManager: "0x401F6c983eA34274ec46f84D70b31C151321188b",
-    baseCrossDomainMessenger: "0x866E82a600A1414e583f7F13623F1aC5d58b0Afa",
-    baseStandardBridge: "0x3154Cf16ccdb4C6d922629664174b904d80F2C35",
-    l1BlastBridge: "0x3a05E5d33d7Ab3864D53aaEc93c8301C1Fa49115",
-    blastCrossDomainMessenger: "0x5D4472f31Bd9385709ec61305AFc749F0fA8e9d0",
-    blastStandardBridge: "0x697402166Fbf2F22E970df8a6486Ef171dbfc524",
+    l1BlastBridge: "0xc644cc19d2A9388b71dd1dEde07cFFC73237Dca8",
```

Comment:
I think this was meant to be deleted?

Reply:
Good catch - it was actually incorrect. Fixed here: https://github.com/across-protocol/contracts/pull/714/commits/4e81f7a0b72c310c80e30242ec5c7ca22acfd529